dipy/tracking/local_tracking.py

```diff
@@ -91,6 +95,7 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
         self.fixed_stepsize = fixedstep
         self.max_cross = max_cross
         self.max_length = maxlen
+        self.min_length = minlen
```

Comment:
just to avoid weird setup for someone not paying attention.
On the top of the `__init__py`, can you add a check, something like (pseudo code to validate):
```python
if minlen > maxlen:
    maxlen = minlen + 2
   warn("max length updated due to strange value")
```